frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,68 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export const Menu: ComponentMultiStyleConfig = {
+  parts: [
+    'buttonBorder',
+    'button',
+    'content',
+    'list',
+    'item',
+    'groupTitle',
+    'command',
+    'divider',
+    'icon',
+  ],
+  baseStyle: {
+    icon: {
+      ml: '1.5rem',
+    },
+    button: {
```

Comment:
i don't think we need theming for `baseStyle.button`; we should just be using Button's own themes